views/stanford/about_get_involved.pug

```diff
@@ -2,29 +2,126 @@ extends ../layout
 
 block styles
   link(rel='stylesheet', href=Config.ASSET_CDN + '/stylesheets/thingpedia.css')
+  link(rel='stylesheet', href=Config.ASSET_CDN + '/stylesheets/get-involved.css')
+  link(rel='stylesheet', href=Config.ASSET_CDN + '/stylesheets/glide.core.min.css')
+  link(rel='stylesheet', href=Config.ASSET_CDN + '/stylesheets/glide.theme.min.css')
+
 
 block page_name
   - stdlayout = false;
-  = _("Contribute To Almond")
+  = _("Get Involved | Almond")
 
 block content
   div#page-body
     div.sections
       section.divider
-        h3= _("Get Involved")
+        h3#jumbotron-page-header= _("Almond, made by you.")
         div.container
-          div.row#dev-program-intro
+          div.row#jumbotron-page-subhead
             div.col-xs-12.col-md-8.col-md-offset-2
-              p= _("Almond is a community project, and we welcome anyone to use it, contribute to it, and help us make it the best virtual assistant.")
-              p= _("You can contribute to Thingpedia, our crowdsourced, non-proprietary repository of digital interfaces and their linguistic representation. You can improve our website, client applications, dialog agents, or even our underlying technology. You can use, remix, and rebrand Almond in your own project.")
-              p
-                strong= _("We welcome everyone to contribute to Almond!")
+              p.text-center.
+                As an #[em open-source] virtual assistant, Almond depends on the work and resources of our community to develop new features, support more devices, and build cool new applications.
+        div.container-fluid
+          div.row
+            div.col-xs-12
+              div#jumbotron-carousel.glide
+                div.glide__track(data-glide-el='track')
+                  ul.glide__slides
+                    li.glide__slide
+                      div.carousel-card(style={ background: 'url(' + Config.ASSET_CDN + '/images/home_assistant.png' + ') no-repeat center', 'background-size': 'cover' })
+                        a(href="https://www.home-assistant.io/", target="_blank")
+                          div.carousel-card-gradient
+                            h3.
+                              Home Assistant
+                            p.carousel-card-subtitle.
+                              Home Assistant, a widely used open-source home automation platform built on Python that allows users to monitor and control their smart devices, 
+                              uses Almond as a language interface to allow its users to control devices using their speech.
+                    li.glide__slide
+                      div.carousel-card(style={ background: 'url(' + Config.ASSET_CDN + '/images/almond_music.jpg' + ') no-repeat center', 'background-size': 'cover' })
```

Comment:
No inline style if we can avoid it (because it prevents us from using a Content-Security-Policy). Put this in the get-involved.css and use a relative URL

Reply:
Ok. Isn't it bad to hard-code the Config.ASSET_CDN in the css?